### PR TITLE
Remove unused resources for EOY

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Based on the End Of Year survey conducted by the wonderful @schwesig, we are removing unused resources and all manifests related to them.

This PR should remove everything from the following projects:
    - boatrockers
    - fybrik-applications
    - ttm-as-a-service
    - ddas-proj
    - chrisproject-dev
    - seraph-platform-mq
    - humair-sandbox (already removed by humair himself)
    - ushift-dev
    - training-model
    - stackrox
    - varangian-test
    - prometheus-anomaly-detector

/cc @tumido, requesting a second set of eyes don't want any breaking changes.
/cc @schwesig I am not sure about the removal of the OCTO apps, I only saw `ushift-dev` and `training-model` listed, although I am not sure if they meant every application in their `octo-ushift-dev` argocd project which would also inclued: `donkey-car`, `keylime`, `cosigned` and the `octo-summit` stuff. Can you touch base with them to figure out if I should remove everything related to OCTO?

### CODE REVIEW

**Comment by schwesig** [comment]
@Gregory-Pereira thank you so much for the work, figuring it out and create this PR.
I will reach out to them and let you know.

**Comment by schwesig** [comment]
@Gregory-Pereira yes, to remove
- donkey-car
- keylime
- cosigned
- octo-summit

talked to @MichaelClifford and @cooktheryan 

also, from @sallyom : ok to remove telemetry / demo projects

File: cluster-scope/bundles/seraph/kustomization.yaml [hunk 1]
```diff
@@ -4,4 +4,3 @@ kind: Kustomization
 resources:
   - ../../base/core/namespaces/seraph-host-inventory
   - ../../base/core/namespaces/seraph-ingress
-  - ../../base/core/namespaces/seraph-platform-mq
```
Comment (tumido) [inline cluster-scope/bundles/seraph/kustomization.yaml:7]: Please remove all the `seraph` namespaces + user group.

**Review by harshad16 — COMMENTED** [review]
/lgtm

:+1:  thanks for the effort

**Review by tumido — CHANGES_REQUESTED** [review]
There's some `seraph` namespaces left floating around. Can you please remove them as well? :slightly_smiling_face:

**Pull request merged**